gen.py

```diff
@@ -72,15 +72,25 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+# pylint: disable=invalid-name
+class X(BaseModel):
+    __root__: dict[FilePath, PlotColumn] = Field(None)
+
+
+# pylint: disable=invalid-name
+class Y(BaseModel):
```

Comment:
Also, how about `PlotFileColumnsMap`/`PlotFileColumnsDict` as above?

Reply:
The only reason that I used `X`/`Y` for names is that they show up as the title of the field in the `onHover` provider. E.g if I change the `X` class to `PlotFileColumnsDict` we get:

<img width="672" alt="image" src="https://user-images.githubusercontent.com/37993418/216284105-0241ab5c-4eb6-4af2-baa8-1cd3a4fd87e8.png">


WDYT?

Reply:
I was thinking of adding `title=""` to `Field`, but that does not seem to work for `__root__`.